### PR TITLE
external deposit/withdrawal links

### PR DESCRIPTION
**Pull request by sunnya97**

### CODE REVIEW

**Comment by dogemos** [comment]
Will need to resolve the new assets that got added, but otherwise looks good to go in preview test.

@jonator

File: src/config.ts [hunk 1]
```diff
@@ -965,13 +971,28 @@ export const IBCAssetInfos: {
...
 		coinMinimalDenom: 'cw20:juno1g2g7ucurum66d42g8k5twk34yegdq8c82858gz0tq2fc75zy7khssgnhjl',
 		ics20ContractAddress: 'juno1v4887y83d6g28puzvt8cl0f3cdhd3y6y9mpysnsp3k8krdm7l6jqgm0rkn',
 	},
+	{
+		counterpartyChainId: 'juno-1',
+		sourceChannelId: 'channel-169',
+		destChannelId: 'channel-47',
+		coinMinimalDenom: 'cw20:juno1g2g7ucurum66d42g8k5twk34yegdq8c82858gz0tq2fc75zy7khssgnhjl',
+		ics20ContractAddress: 'juno1v4887y83d6g28puzvt8cl0f3cdhd3y6y9mpysnsp3k8krdm7l6jqgm0rkn',
+	},
```
Comment (jonator) [inline src/config.ts:987]: Was duplicating this entry intensional?

File: src/pages/assets/AssetBalancesList.tsx [hunk 2]
```diff
@@ -169,45 +171,53 @@ export const AssetBalancesList = observer(function AssetBalancesList() {
...
-										open: true,
-										counterpartyChainId: bal.chainInfo.chainId,
-										currency: currency as IBCCurrency,
-										sourceChannelId: bal.sourceChannelId,
-										destChannelId: bal.destChannelId,
-										isWithdraw: true,
-										ics20ContractAddress: bal.ics20ContractAddress,
-									});
+									if (bal.depositUrlOverride) {
+										window.open(bal.depositUrlOverride, '_blank');
```
Comment (sunnya97) [inline src/pages/assets/AssetBalancesList.tsx:209]:
```suggestion
										window.open(bal.withdrawUrlOverride, '_blank');
```

Comment (JeremyParish69) [reply src/pages/assets/AssetBalancesList.tsx:209]: you suggest to change 209 to `window.open(bal.withdrawUrlOverride, ...` but I believe live 208: `if (bal.deposit...` should also be changed to `if (bal.withdraw...`

Comment (sunnya97) [reply src/pages/assets/AssetBalancesList.tsx:209]: Good catch

File: src/pages/assets/AssetBalancesList.tsx [hunk 3]
```diff
@@ -169,45 +171,53 @@ export const AssetBalancesList = observer(function AssetBalancesList() {
...
-									setDialogState({
-										open: true,
-										counterpartyChainId: bal.chainInfo.chainId,
-										currency: currency as IBCCurrency,
-										sourceChannelId: bal.sourceChannelId,
-										destChannelId: bal.destChannelId,
-										isWithdraw: true,
-										ics20ContractAddress: bal.ics20ContractAddress,
-									});
+									if (bal.depositUrlOverride) {
```
Comment (sunnya97) [inline src/pages/assets/AssetBalancesList.tsx:208]:
```suggestion
									if (bal.withdrawUrlOverride) {
```

File: src/config.ts (hunk 1 above)
Comment (sunnya97) [reply src/config.ts:987]: No

Comment (sunnya97) [inline src/config.ts:987]:
```suggestion
```

File: src/config.ts [hunk 4]
```diff
@@ -2608,7 +2665,11 @@ export const EmbedChainInfos: ChainInfoWithExplorer[] = [
 		],
 		feeCurrencies: [
 			{
+<<<<<<< sunny-external-links
+        coinDenom: 'CRBRUS',
+=======
```
Comment (sunnya97) [inline src/config.ts:2670]:
```suggestion
```

File: src/config.ts [hunk 5]
```diff
@@ -2608,7 +2665,11 @@ export const EmbedChainInfos: ChainInfoWithExplorer[] = [
...
 				coinDenom: 'CRBRUS',
+>>>>>>> master
```
Comment (sunnya97) [inline src/config.ts:2672]:
```suggestion
```

**Comment by jonator** [comment]
@victortokaito Have we finalized the design for external links? I think we'd want that UI here to indicate withdraw/deposit is handled on an external site.

**Comment by sunnya97** [comment]
Going to merge this and move

> the design for external links? I think we'd want that UI here to indicate withdraw/deposit is handled on an external site.

to a new issue.  None of the assets on the main use this, only frontier, so I think it should be okay to get this merged in separately

**Pull request merged**